app/src/main/java/com/hedvig/app/feature/embark/passages/TextActionSetViewModel.kt

```diff
@@ -8,9 +8,28 @@ class TextActionSetViewModel : ViewModel() {
     private val _isValid = MutableLiveData<HashMap<Int, Boolean>>(hashMapOf())
     val isValid: LiveData<HashMap<Int, Boolean>> = _isValid
 
+    private val _inputs = MutableLiveData<Map<Int, String>>()
+    val inputs: LiveData<Map<Int, String>> = _inputs
+
```

Comment:
Would it be possible to pass TextActionSetData as constructor parameter to the view model? I know it was difficult with dagger :(

Reply:
Kind of easy actually: https://insert-koin.io/docs/reference/koin-android/viewmodel#viewmodel-and-injection-parameters

Reply:
Also, I think that use-case is made easier through Hilt 😉

Reply:
Cool! Should be able to pass `parameters: ParametersDefinition` to `Fragment.viewModel()` and then specify the type in the koin module?

Reply:
Yes, there's good potential here, let me investigate 🙂

Reply:
It's glorious!